.github/workflows/test-all-benchmarks-schedule.yml

```diff
@@ -51,6 +51,12 @@ jobs:
         run: |
           ./tools/benchmark-all-runtimes.sh --parallel
 
+      - name: Upload benchmark logs
+        uses: actions/upload-artifact@v4
+        with:
+          name: benchmark-logs
+          path: tmp/benchmark_log_*.txt
```

Comment:
Does this work if the previous step fails? Or does it need if: always()

Reply:
Yes, you are correct, we will miss logs in case of failure.
Added if always. 
Thanks!